ROADMAP.md

```diff
@@ -153,28 +153,49 @@ Any dates listed below and the specific issues that will ship in a given milesto
-* Improved logging and eventing [crossplane/crossplane-runtime#104](https://github.com/crossplane/crossplane-runtime/issues/104)
+## [v0.8.0 Template Stacks simplify cloud-native app and infrastructure provisioning](https://github.com/crossplane/crossplane/releases/tag/v0.8.0)
+- Template Stacks
+  - Template Stacks are enabled default on the Crossplane Helm chart [#1266](https://github.com/crossplane/crossplane/pull/1266)
+  - Helm and kustomize engine support for pluggable rendering
+  - Minimal Stacks ported to use Template Stacks and published to https://hub.docker.com/u/crossplane
+    - Spin up secure cloud environments (nets, subnets, secure service connectivity, k8s clusters, resource classes, etc.) with just a few lines of yaml (single CR)
+    - [stack-minimal-gcp](https://github.com/crossplane/stack-minimal-gcp/pull/1)
+    - [sample-stack-wordpress](https://github.com/crossplane/sample-stack-wordpress/pull/31)
+    - Minimal Stacks for ready-to-run cloud environments (GCP, AWS, Azure) [#1136](https://github.com/crossplane/crossplane/issues/1136)
+- Support for PostgreSQL 11 on the `PostgreSQLInstance` claim (thanks first-time contributor @vasartori!) [#1245](https://github.com/crossplane/crossplane/pull#1245)
+- RBAC group and role refinements [#1195](https://github.com/crossplane/crossplane/pull/1195) and [#1219](https://github.com/crossplane/crossplane/pull/1219)
+- Enable [stack-aws](https://github.com/crossplane/stack-aws) to authenticate to the AWS API using [IAM Roles for Service Accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) when running on EKS [stack-aws#126](https://github.com/crossplane/stack-aws/pull/126)
+- Improved logging and eventing [crossplane/crossplane-runtime#104](https://github.com/crossplane/crossplane-runtime/issues/104)
+  - New [logging framework](https://github.com/crossplane/crossplane-runtime/pull/108) from `crossplane-runtime` in Crossplane controllers [#1227](https://github.com/crossplane/crossplane/pull/1227)
+- Support default select values in the UI schema for Crossplane resources [#1253](https://github.com/crossplaneio/crossplane/pull/1253)
+- Observability guide to assist stack developers with logging and eventing in their controllers [#1236](https://github.com/crossplaneio/crossplane/pull/1236)
+- Host-aware Stack Manager [#1038](https://github.com/crossplane/crossplane/issues/1038)
+  - Enables deploying multiple Crossplane instances watching different Kubernetes API servers (i.e. multiple API servers running in different namespaces) on a single Host Kubernetes cluster.
+
+## v0.9.0
+* Incorporate versioning and upgrade design feedback [#1160](https://github.com/crossplane/crossplane/issues/1160)
+* Rename GitHub org from crossplaneio to crossplane
+* Support Crossplane as an OAM runtime using the revised 'Kubernetes-friendly` OAM spec
```

Comment:
I think there may be a formatting bug here with ' and `

Reply:
fixed